### ISSUE TITLE
Gcc build broken on Ubuntu Focal

### ISSUE THREAD

**Issue by andrewbird** [FDOS/kernel#27]
It seems that the version of NASM that comes with Ubuntu Focal 20.04 now cares that there is a duplicate output file specified. I would have tried to fix it, but the first `-o iasmsupt.obj` appears to come from the NASMFLAGS variable and I couldn't see where that got set?

~~~
nasm -Dgcc -D_INIT -f obj -DWITHFAT32 -i../hdr/ -DXCPU=86 -felf -o iasmsupt.obj -o iasmsupt.obj asmsupt.asm
iasmsupt.obj: fatal: more than one output file specified: iasmsupt.obj

make[1]: *** [GNUmakefile:161: iasmsupt.obj] Error 1
make[1]: Leaving directory '/clients/common/fdkernel.git/kernel'
~~~

**Comment by andrewbird** [comment]
Here's the Travis build against Focal for reference https://travis-ci.com/github/andrewbird/kernel/jobs/399461162

**Comment by andrewbird** [comment]
Now fixed by #28
